assets/responsibleai/environments/responsibleai-vision-py38/context/conda_dependencies.yaml

```diff
@@ -7,7 +7,8 @@ channels:
 dependencies:
   - python=3.8
   - pip
-  - pytorch
+  # lower bound on pytorch due to CVE https://github.com/advisories/GHSA-47fc-vmwq-366v
```

Comment:
Shall we move this comment on the PR description? Seems needless as we can track the PR from git blame.

Reply:
I removed the line. PR description already has the link.